Client/Frontend/Browser/Playlist/Controllers/PlaylistMoveFolderView.swift

```diff
@@ -0,0 +1,269 @@
+        }
+        
+        init(favIcon: PlaylistItem) {
+            guard let pageSrc = favIcon.pageSrc,
+                  let favIconUrl = URL(string: pageSrc) else {
+                image = nil
+                return
+            }
+            
+            loadImage(url: favIconUrl, isFavIcon: true)
+        }
+        
+        private func loadImage(url: URL, isFavIcon: Bool) {
+            renderer.loadThumbnail(assetUrl: isFavIcon ? nil : url,
+                                   favIconUrl: isFavIcon ? url : nil,
+                                   completion: { [weak self] image in
+                self?.image = image
+            })
+        }
+    }
+}
+
+private struct PlaylistFolderView: View {
+    let iconColor: Color
+    let titleColor: Color
+    let folder: PlaylistFolder?
+    @Binding var selectedFolder: PlaylistFolder?
+    
+    var body: some View {
+        HStack(spacing: 18.0) {
+            Image(systemName: "folder")
+                .renderingMode(.template)
+                .aspectRatio(contentMode: .fit)
+                .foregroundColor(iconColor)
+            
+            VStack(alignment: .leading) {
+                if let folder = folder {
+                    let itemCount = folder.playlistItems?.count ?? 0
+                    
+                    Text(folder.title ?? "")
+                        .font(.body)
+                        .foregroundColor(titleColor)
+                    Text("\(itemCount) \(itemCount == 1 ? Strings.PlaylistFolders.playlistFolderSubtitleItemSingleCount : Strings.PlaylistFolders.playlistFolderSubtitleItemCount)")
+                        .font(.footnote)
+                        .foregroundColor(titleColor)
+                } else {
+                    Spacer()
+                }
+            }
+            
+            if selectedFolder?.uuid == folder?.uuid {
+                Spacer()
+                Image(systemName: "checkmark")
+                    .renderingMode(.template)
+                    .aspectRatio(contentMode: .fit)
+                    .foregroundColor(.white)
+            }
+        }
+        .padding(EdgeInsets(top: 7.0, leading: 0.0, bottom: 7.0, trailing: 0.0))
+    }
+}
+
+struct PlaylistMoveFolderView: View {
+    @FetchRequest(
+        entity: PlaylistFolder.entity(),
+        sortDescriptors: [
+            NSSortDescriptor(keyPath: \PlaylistFolder.order, ascending: true),
+            NSSortDescriptor(keyPath: \PlaylistFolder.dateAdded, ascending: false)
+        ],
+        predicate: NSPredicate(format: "uuid != %@", PlaylistFolder.savedFolderUUID)
+    ) var folders: FetchedResults<PlaylistFolder>
+    
+    @FetchRequest(
+        entity: PlaylistFolder.entity(),
+        sortDescriptors: [],
+        predicate: NSPredicate(format: "uuid == %@", PlaylistFolder.savedFolderUUID)
+    ) var savedFolder: FetchedResults<PlaylistFolder>
+    
+    @State private var moveDisabled: Bool = true
```

Comment:
This shouldn't be `State`, the `selectedFolder` changing is the actual event that will trigger a body recomputation, this should be 

```swift
private var isMoveDisabled: Bool { 
  selectedFolder == PlaylistManager.shared.currentFolder // or w/e equality is based off of here
}
```